### ISSUE TITLE
Adjust the outputs to reflect states without state income tax 

### ISSUE THREAD

**Issue by noman404** [PolicyEngine/policyengine-taxsim#166]
As special cases are growing and more states requires to skip variable calls, they needs to be optimized and generalized. As discussing [here ](https://github.com/PolicyEngine/policyengine-taxsim/pull/155)

**Comment by PavelMakarchuk** [comment]
Is this geared towards states without state income taxes - if so we should name the issue that way

**Comment by noman404** [comment]
Right, this to more to cover the special cases for the states does not have that as well as if any other scenarios related to the state related calculation

**Comment by PavelMakarchuk** [comment]
> Right, this to more to cover the special cases for the states does not have that as well as if any other scenarios related to the state related calculation

Lets keep this issue just to states without state income tax for simplicity

**Comment by MaxGhenis** [comment]
>as if any other scenarios related to the state related calculation

such as?

**Comment by noman404** [comment]
For now they are TX, FL, NV and also the n/a cases in excel list which are not implemented right now.
